docs/presto-guide.md

```diff
@@ -220,7 +220,8 @@ Now you can use the Presto CLI on the coordinator to query data sources in the c
   Secondly: Create a folder named 'carbondata' under $PRESTO_HOME$/plugin and
   copy all jars from carbondata/integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT
         to $PRESTO_HOME$/plugin/carbondata
-
+  **NOTE:**  Not copy one assemble jar, need to copy many jars from integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT
```

Comment:
```suggestion
  **NOTE:**  Don't just copy assemble jar, make sure to copy all jars from integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT
```

Reply:
How about : Not copy the assemble jar, make sure to copy all jars ...

Reply:
@chenliang613  "Not copy" is oral， suggest  “No need to copy assemble jar” or “”assemble jar is not required“”

Reply:
Why is this NOTE needed? I think line#221 has already made it clear.

Reply:
Is it the folder name can not draw users' attention? I found that a ZIP file is created also. What about asking user to extract the jars in the zip file to a folder under presto

Reply:
> Why is this NOTE needed? I think line#221 has already made it clear.

Many community guys made mistakes, this is why add one more note.